examples/active_plasma_lens/plot_APL_ChrPlasmaLens_analytical.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022-2025 ImpactX contributors
+# Authors: Axel Huebl, Chad Mitchell, Kyrre Sjobak
+# License: BSD-3-Clause-LBNL
+#
+# -*- coding: utf-8 -*-
+
+## Plots analytical Twiss parameters
+#  as a function of APL gradient [T/m]
+
+import matplotlib.pyplot as plt
+import numpy as np
+from run_APL_ChrPlasmaLens import analytic_final_estimate
```

Comment:
```suggestion
from run_APL import analytic_final_estimate
```

Reply:
Just caught this - name change.  I just tried it.

Reply:
Thank you!